phoenix-assembly/src/build/components/all-common-dependencies.xml

```diff
@@ -28,6 +28,8 @@
       <unpack>false</unpack>
       <outputDirectory>/lib</outputDirectory>
       <includes>
+        <include>org.apache.phoenix:phoenix-hbase-compat-${hbase.compat.version}</include>
```

Comment:
This is to include the user provided compat jar in tarball right?

Reply:
Exactly.
This may be needed by things like phoenix-pherf or the tracing server.

Reply:
Though IIRC pherf already uses phoenix-client, so it may not be.
But we are shipping so much junk in /lib, that one more jar doesn't make a difference.

Reply:
I think this is fine.